INSTALL.md

```diff
@@ -109,7 +109,15 @@ be edited here.
         ### if you used approach 1, log out; you should be yourself now
         $ exit
 
-1. Generate tables in postgres
+1. Generate tables in postgres and run migrations
+
+        ### Run the initial sql file (created before migrations were being used)
```

Comment:
If this line just does schema setup (i.e., no data), then I think it would be good to say so here.  It's good for the developer to have a rough idea of what's going on in each step.